### ISSUE TITLE
Use javafx 17 modules

### ISSUE THREAD

**Issue by BenHerbst** [jruby/jrubyfx#128]
I tired with openjdk and javafx system variables set on windows to use this with javafx 17, and also tried to use the zulu jdk bundled with javafx without success. I am using jrubyfx 2.0.0 and I am completly new to jruby, only used java and kotlin with javafx before.
Error:
JavaFX runtime not found.  Please install Java 7u6 or newer or set environment variable JFX_DIR to the folder that contains jfxrt.jar
If you have Java 7u6 or later, this is a bug. Please report to the issue tracker on github. Include your OS version, 32/64bit, and architecture (x86, ARM, PPC, etc)

**Comment by byteit101** [comment]
Some questions:

1. What version of JRuby?
2. What Architecture?
3. Is the `JAVA_HOME` environment variable set and pointing to the same java version on the path?
4. Are you using a ruby version manager?
5. If not, are you using the "complete" jruby jar directly?
6. If using the "complete" jar, are `GEM_HOME` and `GEM_PATH` set?

Using JRuby 9.3.3 against JRubyFX master on x86_64 Linux with the java build listed below worked just fine for me

I've asked @enebo if he has the same issue on Windows.


```
~/all-the-jdks$ ./with-java zulu17
zulu17 ~/all-the-jdks$ java -version
openjdk version "17.0.2" 2022-01-18 LTS
OpenJDK Runtime Environment Zulu17.32+13-CA (build 17.0.2+8-LTS)
OpenJDK 64-Bit Server VM Zulu17.32+13-CA (build 17.0.2+8-LTS, mixed mode, sharing)
zulu17 ~/all-the-jdks$ cd ~/projects/jrubyfx/samples/javafx 
zulu17 ~/projects/jrubyfx/samples/javafx$ jruby -I ~/projects/jrubyfx/lib hello_jrubyfx.rb 
zulu17 ~/projects/jrubyfx/samples/javafx$ jruby -I ~/projects/jrubyfx/lib line_chart.rb 
```
If you install the gem you shoudn't need the `-I`

**Comment by enebo** [comment]
I was able to:

  1. jruby -S rake reflect
  2. jruby -S gem install pkg/jrubyfx*.gem (rdoc error did occur but it did not prevent the gem from installing)
  3. jruby analog_clock.rb (from samples)

I am on windows 10 still and I install Zulu SDK FX to test this.  I originally somehow got just SDK and not SDK FX and I would swear I got the right one.  I also forgot 2.0 was not released and used the very old in the tooth gem from rubygems (which does not work).  I got there in the end though.

@BenHerbst Just to make absolutely sure.  If you run java --list-modules you do see the javafx modules in there?
